image_geometry/CMakeLists.txt

```diff
@@ -22,9 +25,10 @@ add_library(${PROJECT_NAME}
   src/pinhole_camera_model.cpp src/stereo_camera_model.cpp
 )
 target_link_libraries(${PROJECT_NAME} ${OpenCV_LIBRARIES})
-ament_target_dependencies(${PROJECT_NAME}
-  "OpenCV"
-  "sensor_msgs"
+ament_target_dependencies(
+  ${PROJECT_NAME}
+  OpenCV
+  sensor_msgs
```

Comment:
Does this actually change anything?

Reply:
No.

Reply:
I thought the preferred style was without quotes, is that true?

Reply:
Historically, I don't think so, but maybe the trend has changed 😅

Reply:
Really doesn't matter though.